navigation/create_control/config/create_2_control.yaml

```
@@ -0,0 +1,43 @@
+# http://wiki.ros.org/diff_drive_controller for more parameters.
+
+mobile_base_controller:
+  type        : "diff_drive_controller/DiffDriveController"
+  left_wheel  : 'wheel_left_joint'
+  right_wheel : 'wheel_right_joint'
+  publish_rate: 50.0               # default: 50
+  pose_covariance_diagonal : [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
+  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
+
+  # Wheel separation and diameter. These are both optional.
+  # diff_drive_controller will attempt to read either one or both from the
+  # URDF if not specified as a parameter
+  wheel_separation : 1.0
+  wheel_radius : 0.3
```

I took the values from [here](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_description/urdf/create_2.xacro#L8-L11).

```suggestion
  wheel_separation : 0.235
  wheel_radius : 0.036
```

---

Changed